AK/StringView.cpp

```diff
@@ -75,6 +75,32 @@ Vector<StringView> StringView::split_view(const char separator, bool keep_empty)
     return v;
 }
 
+Vector<StringView> StringView::split_view(const StringView& separator, bool keep_empty) const
+{
+    if (is_empty())
+        return {};
+
+    ASSERT(!separator.is_empty());
```

Comment:
I'd move the assert above

Reply:
Ehhhhhhhhhhhhhhhhhhh, I can't think of a reason not to, sure.